build.sbt

```diff
@@ -119,3 +123,42 @@ lazy val shredder = project.in(file("modules/shredder"))
     )
   )
   .dependsOn(common)
+  .enablePlugins(BuildInfoPlugin)
+
+lazy val streamShredder = project.in(file("modules/stream-shredder"))
+  .settings(
+    name        := "snowplow-rdb-stream-shredder",
+    description := "Stream Shredding job",
+    buildInfoPackage := "com.snowplowanalytics.snowplow.rdbloader.shredder.stream.generated",
+    buildInfoKeys := List(name, version, description),
+    addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1"),
+    scalacOptions.in(Compile) ~= {    // TODO: remove before production
```

Comment:
Will this TODO get fixed before release?

Reply:
Yup, removed it, thanks for a nudge.